assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -233,7 +233,7 @@ const SelectionCardContainer = ({
   return (
     <div
       {...keepUserInputFromLeaflet}
-      className="m-map-display__selected-entity-card-container"
+      className="c-map-selection-card-container"
```

Comment:
c-map-display is not an existing block, so establishing `c-map-selection-card-container` as a block